https://issues.redhat.com/browse/ACM-17893 Move cert doc (#7733)
open-cluster-management/rhacm-docs: secure_cluster/cert_byo.adoc

oc -n open-cluster-management-observability create secret tls observability-client-ca-certs --cert ./clientCACert.pem --key ./clientCAKey.pem
----

[#replace-cert-rbac]
== Replacing certificates for _rbac-query-proxy_ route

You can replace certificates for the `rbac-query-proxy` route. See xref:../secure_cluster/cert_byo.adoc#openssl-commands-ca[Generating CA certificates by using OpenSSL commands] to create certificates.

When you create a Certificate Signing Request (CSR) by using the `csr.cnf` file, update the `DNS.1` field in the `subjectAltName` section to match the host name of the `rbac-query-proxy` route.

Complete the following steps:

> . Run the following command to retrieve the host name:

Reviewer: I'd switch around all the steps like this:
```suggestion
. Retrieve the host name by running the following command:
```

If you're just skimming through the docs and looking for one particular step, it's easier to find what you're looking for.

+
[source,bash]
----
oc get route rbac-query-proxy -n open-cluster-management-observability -o jsonpath="
{.spec.host}"
----

> . Run the following command to create a `proxy-byo-ca` secret by using the generated certificates:

Reviewer: This one could be:
```suggestion
. Create a `proxy-byo-ca` secret by using the generated certificates. Run the following command:
```
